src/ui/embeds/figma/figma-embed.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import classNames from 'classnames';
+import styles from './figma-embed.module.scss';
+
+export type FigmaEmbedProps = {} & React.IframeHTMLAttributes<HTMLIFrameElement>;
+
+export function FigmaEmbed({ loading = 'lazy', width = '100%', height = '600', className, ...rest }: FigmaEmbedProps) {
+  return (
+    <iframe
+      {...rest}
+      className={classNames(styles.iframe, className)}
+      width={width}
+      height={height}
```

Comment:
can you use the `className` for width and height as well?

Reply:
why would you want to? The width and height allows you to easily modify the height for example for different embeds which is useful if you havea  component that has a longer figma file for example so it is a very useful prop that you might want to modify depending on where you are showing the figma embed or for certain designs. having it in classnames so in the styles means less flexability

Reply:
you can override it how ever you want in the future. they are passed via the `...rest` prop. 
but for your default styles you should do it with the class. 
- it is easier to override (has lower specificity)
- you already pass a className so it might as well be in there.
- less code

Reply:
after pair programming it was decided to leave this as props as when using the css the width and height is not overriding it.